hlf/scala/admission.md

```diff
@@ -0,0 +1,73 @@
+# Hyperledger Scala API Admission
+
+In the following, the hlf-api for the "admission" handling will be explained.
+For insight in the General hlf-api, please refer to it's .Readme - File.
+> [!NOTE]
+Any Method can throw these [Exceptions](errors.md#Exceptions).
+
+For "admission" handling we offer the following interface. 
+Additionally, as described in [General Communication](general-communication.md), different proposal calls can be made.
+
+
+## AddAdmission(enrollmentId: String, courseId: String, moduleId: String, timestamp: Option[String])
+- Returns
+    - Admission :: Json (refer to: [Admission](../chaincode/admission.md#Admission))
+        - => Success
+    - TransactionError :: Json (refer to: [DetailedError](../chaincode/errors.md#DetailedError))
+        - => error is returned
+          - If the given parameters could not be parsed.
+    - TransactionError :: Json (refer to: [SemanticError](../chaincode/errors.md#SemanticError))

```

Comment:
This comment is here to remind us that there is a SemanticError now.

Reply:
, which may be removed again. (refer to discussion in chaincode related PR)

Reply:
Semantic error was removed